sw/device/lib/testing/random.h

```diff
@@ -0,0 +1,30 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_RANDOM_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_RANDOM_H_
+
+#include <stdint.h>
+
+/**
+ * Generate a pseudo-random unsigned integer.
+ *
+ * For testing purposes, a simple LFSR generates random numbers starting with
+ * a known seed.
+ * @return A pseudo-random 32-bit value.
+ */
+uint32_t urandom(void);
```

Comment:
I think this probably deserves a longer name, because short names are kind of a bad idea in C. Maybe consider names like:
- `random_gen32`

(Same for the range version below.)